packages/vue/src/components/reset-password.vue

```diff
@@ -1,28 +1,46 @@
 <template>
   <slot v-bind="$attrs" name="resetPasswordSlotI">
-    <base-wrapper v-bind="$attrs" data-amplify-wrapper>
-      <base-form
-        data-amplify-authenticator-resetpassword
-        @submit.prevent="onResetPasswordSubmit"
-        @change="onChange"
-      >
-        <base-heading>
+    <base-form
+      v-bind="$attrs"
+      data-amplify-authenticator-resetpassword
+      @submit.prevent="onResetPasswordSubmit"
+      @change="onChange"
+    >
+      <base-wrapper class="amplify-flex" style="flex-direction: column">
+        <base-heading class="amplify-heading" :level="3">
           {{ resetPasswordHeading }}
         </base-heading>
-        <base-field-set :disabled="actorState.matches('resetPassword.pending')">
-          <base-label data-amplify-resetpassword-label>
-            <base-text>Username</base-text>
-            <base-input
-              name="username"
-              :placeholder="enterUsernameText"
-              autocomplete="username"
-              required
-              type="username"
-            ></base-input>
-          </base-label>
+        <base-field-set
+          class="amplify-flex"
+          style="flex-direction: column"
+          :disabled="actorState.matches('resetPassword.pending')"
+        >
+          <base-wrapper
+            class="amplify-flex amplify-field amplify-textfield"
+            style="flex-direction: column"
+          >
+            <base-label class="amplify-label sr-only" for="amplify-field-7dce">
+              Username
+            </base-label>
+            <base-wrapper class="amplify-flex">
+              <base-input
+                class="amplify-input amplify-field-group__control"
+                id="amplify-field-7dce"
+                aria-invalid="false"
+                name="username"
+                :placeholder="enterUsernameText"
+                autocomplete="username"
+                required
+                type="username"
+              ></base-input>
+            </base-wrapper>
+          </base-wrapper>
         </base-field-set>
 
-        <base-footer>
+        <base-footer
+          class="amplify-flex"
+          style="flex-direction: column; align-items: unset"
+        >
           <template #footert="{ slotData }">
             <slot
               name="footer"
```

Comment:
After every refactor I'm trying to save the slots I setup originally. I left this one here, in case someone wants to replace the footer on this page.